This fixes the PLATFORM_CSBASE address (only tested on MB411+DB641).

Signed-off-by: Giuseppe Cavallaro <peppe.cavallaro@st.com>

--- linux/drivers/net/smsc_911x/st40.c.orig	2007-11-29 09:01:00.622702000 +0100
+++ linux/drivers/net/smsc_911x/st40.c	2007-11-29 09:04:10.753588000 +0100
@@ -65,11 +65,11 @@
 /* db641 STEM card plugged into mb376 */
 #ifdef DB641_USE_PORT0
 /* STEM CS0 = BankB, A23=0 */
-#define PLATFORM_CSBASE		(0xA1000000UL)
+#define PLATFORM_CSBASE		(0x01000000UL)
 #define PLATFORM_IRQ		(12UL)
 #else
 /* STEM CS1 = BankB, A23=1 */
-#define PLATFORM_CSBASE		(0xA1800000UL)
+#define PLATFORM_CSBASE		(0x01800000UL)
 #define PLATFORM_IRQ		(11UL)
 #endif
 #elif defined(CONFIG_SH_ST_MB519)
@@ -77,11 +77,11 @@
 #ifdef DB641_USE_PORT0
 /* STEM CS0 = BANK1 and A23=0 */
 /* Make sure you read the comment about bodges in mb519/stx7200mboard_init_irq */
-#define PLATFORM_CSBASE		(0xA2000000UL)
+#define PLATFORM_CSBASE		(0x02000000UL)
 #define PLATFORM_IRQ		(MUXED_IRQ_BASE+4)
 #else
 /* STEM CS1 = BANK4 */
-#define PLATFORM_CSBASE		(0xA2800000UL)
+#define PLATFORM_CSBASE		(0x02800000UL)
 #define PLATFORM_IRQ		(MUXED_IRQ_BASE+4)
 #endif
 #elif defined(CONFIG_SH_HMS1)
